source/command-monitoring/tests/unified/redacted-commands.yml

```diff
@@ -213,3 +213,36 @@ tests:
           - commandStartedEvent:
               commandName: isMaster
               command: { isMaster: { $$exists: false } }
+
+  - description: "hello without speculative authenticate is not redacted"
+    operations:
+      - name: runCommand
+        object: *database
+        arguments:
+          commandName: hello
+          command:
+            hello: "public"
```

Comment:
IIRC, the server never checks the _value_ of these fields; however, I'm not sure why we'd use `"public"` here when the documented value is `1`.

 * https://docs.mongodb.com/manual/reference/command/hello/
 * https://docs.mongodb.com/manual/reference/command/isMaster/

Reply:
Yeah I believe most of these authentication-related commands use `1` as a value normally. I used `public` just to emphasize that `hello`'s value will be visible in the published commands (as opposed to `private` in the redacted tests above).

Reply:
I just noticed you were using `private` in other tests when syncing these with PHP. SGTM.